packages/core/src/types.ts

```diff
@@ -2,7 +2,7 @@ import type React from 'react';
 
 export type { PackageJson } from 'type-fest';
 
-export type Format = 'esm' | 'cjs' | 'dts';
+export type Format = 'cjs' | 'esm' | 'dts' | 'dtsm';
```

Comment:
```suggestion
export type Format = 'cjs' | 'esm' | 'dts' | 'dmts';
```

Reply:
I've updated the description with the explanation for `dtsm`.

> I chose `dtsm` as the new format name because it's a [portmanteau](https://en.wikipedia.org/wiki/Portmanteau) of `dts` and `esm`.

Reply:
Ah, I guess this is a "file format" rather than a file extension. Fair enough. Though it was just a typo.